support-frontend/app/services/RecaptchaService.scala

```diff
@@ -26,6 +28,9 @@ class RecaptchaService(wsClient: WSClient)(implicit ec: ExecutionContext) {
       .withMethod("POST")
       .execute()
       .attemptT
-      .leftMap(_.toString)
+      .leftMap(err => {
+        SafeLogger.error(scrub"Recaptcha failed on ${err.toString}")
+        err.toString
+      })
```

Comment:
another option to save on brackets (I don't have a strong preference but it's what I normally do)
```suggestion
      .leftMap { err => 
        SafeLogger.error(scrub"Recaptcha failed on ${err.toString}")
        err.toString
      }
```